README.md

```diff
@@ -637,40 +638,9 @@ EXAMPLES
 
 ## `sf run function start container`
```

Comment:
I notice the command still exists in order to inform the user that it has been deprecated. Maybe we should note that in the README? Or is this autogenerated?

Reply:
README is auto-generated - there is probably a way for me to add something here if you thought it was worth it. I'm fairly unfamiliar with all the CLI bits 😄